### DISCUSSION TITLE
hello

### DISCUSSION THREAD

**Discussion by zalou-215** [community/community#36329] [Other Feature Feedback, Questions, & Ideas]
Hi guys I would like to know how we can add two field in Django models class-based view. Each field is a note so how could i perform calculations adding two fields and dividing the sum by 2 ...

**Comment by ossanna16** [comment]
Hi @zalou-215,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Django. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by Ivanrputra** [comment]
you can overide the save function on models class to fill the field as you want
